quiz.js

```diff
@@ -0,0 +1,100 @@
+//char = character displayed/storred in array
```

Comment:
the spacing in this file is kinda all over the place which makes it hard to read.  Can you go back through and do the proper indentation on each line?  I am happy to help you with this tonight if you are unsure about how things should be indented.